back/node_watcher/src/tasks/createProposal.ts

```diff
@@ -48,22 +48,28 @@ const createProposal: Task<NomidotProposal[]> = {
 
             return {
               ...prev,
-              [type]: curr.toString(),
+              [type]: curr.toJSON(),
             };
           },
           {}
         );
+        if (!proposalRawEvent.PropIndex && proposalRawEvent.PropIndex !== 0) {
```

Comment:
In this case `proposalRawEvent.PropIndex === undefined` (or `isUndefined` from `@polkadot/util`) might be easier to read

Reply:
Much more readable indeed, but this leaves the `null` and empty string aside. I'd rather be extra safe and just check for this 0 exception.